tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -25,3 +25,20 @@ Given('user {string} has shared file/folder {string} with user {string}', functi
       }
     })
 })
+
+When('the user types {string} in the share-with-field', function (input) {
+  return client.page.filesPage().enterAutoComplete(input)
+})
+
+Then('all users and groups that contain the string {string} in their name should be listed in the autocomplete list on the webUI', function (pattern) {
+  return client.page.filesPage().sharingAutoCompleteShallContainElementsWith(pattern)
+})
+
+Then('the users own name should not be listed in the autocomplete list on the webUI', function () {
+  // TODO: where to get the current user from
+  return client.page.filesPage().sharingAutoCompleteShallNotContainElementsWith('admin')
```

Comment:
We have to implement something similar to this https://github.com/owncloud/core/blob/master/tests/acceptance/features/lib/OwncloudPage.php#L284
And remove this hard-coded name.

That can be done in other PR

Reply:
Shouldn't the current logged in user be stored in some variable somewhere?

Reply:
Current admin username is stored in `client.globals.backend_admin_username` but,
That doesn't work for all users.

This step should work for all users. So, we should get the current user using the webUI.
But In core in some API tests we set the current user using 
the behat step `"As user "<user_name>"` and set username as a global variable.
https://github.com/owncloud/core/blob/master/tests/acceptance/features/bootstrap/BasicStructure.php#L676

But I don't think this is a good idea for a webUI test. Because when the user logs out we have to remember to reset this variable.